portal/src/styles/variable.scss

```diff
@@ -1,3 +1,10 @@
 $box-shadow-color: rgba(0, 0, 0, 0.1);
 $widget-box-shadow: 0 0.5px 1px $box-shadow-color,
   0 1.5px 3.5px $box-shadow-color;
+
+@mixin widget-header-text {
```

Comment:
Again, if we are going to reuse a UI component, make it a React component instead of sharing style via `variable.scss`.